units/CHANGELOG.md

```diff
@@ -1,8 +1,29 @@
-# Unreleased
+# 1.0.0 - 2025-02-24
 
-- TODO: Make a comment about `Amount::MAX_MONEY` including breaking serde
+BOOM! A long time in the making but here goes, our first 1.0 crate release.
```

Comment:
I think this is confusing, since it looks like proper 1.0, not alpha. It needs to be clearly marked as such.

Reply:
It looks like false marketing. Someone's going to read it a be like, "awesome, I'm going to update" only to be disappointed.

Reply:
In the light of recent discussion. I now consider this much less concerning than before. Also `units` with no amounts might as well be `rc` is you want. :) Note that I would still like having "alpha" (or "release candidate") written on top of changelog for the time being but I'm not going to block PR over it.